docs/docs/recipes.md

```diff
@@ -154,62 +437,27 @@ By targeting the HTML `body` element, your font will apply to most text on the p
 
 ## Creating layouts
 
-To wrap pages with layouts, use normal React components.
-
-- Walk through creating a layout component in [tutorial part three](/tutorial/part-three/#your-first-layout-component)
-- Gatsby v1 approached layouts differently. If the context is helpful, learn about the [differences in v2](/blog/2018-06-08-life-after-layouts/)
-
-## Deploying
-
-Showtime. Once you are happy with your site, you are ready to go live with it!
-
-### Preparing for deployment
-
-#### Prerequisites
-
-- A [Gatsby site](/docs/quick-start)
-- The [Gatsby CLI](/docs/gatsby-cli) installed
-
-#### Directions
-
-1. Stop your development server if it is running (`Ctrl + C` on your command line in most cases)
-
-2. For the standard site path at the root directory (`/`), run `gatsby build` using the Gatsby CLI on the command line. The built files will now be in the `public` folder.
-
-```shell
-gatsby build
-```
-
-3. To include a site path other than `/` (such as `/site-name/`), set a path prefix by adding the following to your `gatsby-config.js` and replacing `yourpathprefix` with your desired path prefix:
-
-```js:title=gatsby-config.js
-module.exports = {
-  pathPrefix: `/yourpathprefix`,
-}
-```
+Starters are boilerplate Gatsby sites maintained officially, or by the community.
```

Comment:
One more thing -- for some reason this is under `## Creating layouts` and it should be under `## Working with starters` (right below).

Not sure if there was other content you wanted to add for `## Creating layouts`?

Reply:
I pushed this change to your branch (https://github.com/gatsbyjs/gatsby/pull/15620/commits/15aa456e3254f836f09defcab9e61124deab4df7), and went ahead and added the "Working with Themes" recipe in this PR as well (https://github.com/gatsbyjs/gatsby/pull/15620/commits/18178a967cfdc960ca5c2b2959ba169a0ec2821e).

Reply:
Thanks! That's a good place to start for themes - ultimately we'll want it to follow the new format though: https://www.gatsbyjs.org/contributing/docs-templates/#recipes

Reply:
This is challenging. Should it be split into "Using a theme" and "Building a theme"?

Reply:
merging as-is, will address later.